config.json

```diff
@@ -1571,6 +1678,56 @@
       "name": "Basics",
       "slug": "basics",
       "uuid": "5d2b4677-1850-4ee4-890d-dc55a1654626"
+    },
+    {
+      "name": "Booleans",
+      "slug": "booleans",
+      "uuid": "e3942921-34ec-41eb-a999-a5d85a6623fd"
+    },
+    {
+      "name": "Comments",
+      "slug": "comments",
+      "uuid": "9ee93eab-b413-442c-b7fb-76bf044eea2d"
+    },
+    {
+      "name": "Conditionals If",
+      "slug": "conditionals-if",
+      "uuid": "b3731323-f5bb-4b38-931c-2d734667bf8d"
+    },
+    {
+      "name": "Conditionals Switch",
+      "slug": "conditionals-switch",
+      "uuid": "6a4103e0-e7a9-427e-b1bd-716870580c45"
+    },
+    {
+      "name": "Constants",
+      "slug": "constants",
+      "uuid": "b56f68de-44ef-4bd4-8b14-2c50b2a366dd"
+    },
+    {
+      "name": "Errors",
+      "slug": "errors",
+      "uuid": "a960da78-c940-4725-8069-929f42564cdb"
+    },
+    {
+      "name": "Iteration",
+      "slug": "iteration",
+      "uuid": "e18ca160-bbb8-4cd0-882a-0e67f1e60b47"
+    },
+    {
+      "name": "Maps",
+      "slug": "maps",
+      "uuid": "625aa065-b006-4fc5-b276-28e048376b46"
+    },
+    {
+      "name": "Methods",
+      "slug": "methods",
+      "uuid": "d25c6e6b-c6d4-40ce-80f9-cecbc5a72254"
+    },
+    {
+      "name": "Structs",
+      "slug": "structs",
+      "uuid": "5bd431cf-c30d-4f5c-a16d-c62e51720fbd"
```

Comment:
I generated new uuids for everything I'm adding. Is that correct? I saw that there's some `config.orig.json` file around 🤔

Reply:
I agree, new UUIDs are preferable and I don't _think_ that will cause any issues with the website? I think we can try it and see, and revert to the ones in `config.orig.json` as needed. I think we can also remove that file now that you've migrated the exercises over?